apps/hmc/hmc-judicial-payment-service/hmc-judicial-payment-service.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: hmc-judicial-payment-service
+spec:
+  releaseName: hmc-judicial-payment-service
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      #image: hmctspublic.azurecr.io/hmc/hmc-judicial-payment-service: #{"$imagepolicy": "flux-system:hmc-judicial-payment-service"}
```

Comment:
why are you commenting out the image, you need to supply one

Reply:
Supplied image